ci-operator/step-registry/openshift/e2e/openstack/csi/openshift-e2e-openstack-csi-workflow.yaml

```diff
@@ -9,8 +9,7 @@ workflow:
     - chain: ipi-openstack-post
     env:
       OPENSTACK_EXTERNAL_NETWORK: "external"
-      OS_CLOUD: "openstack-cloud"
-      OPENSTACK_COMPUTE_FLAVOR: "m1.s2.xlarge"
+      OPENSTACK_INSTANCE_FLAVOR: "m1.s2.xlarge"
       CONFIG_API_VERSION: "v1beta4"
       CONFIG_TYPE: "minimal"
       BASE_DOMAIN: shiftstack.devcluster.openshift.com
```

Comment:
all of these values maybe removed since those are the default values, however i thought best live them here until the owners of the workflows verify they don't want them here.